FastSimulation/Configuration/test/run_TrackingOnly_cfg.py

```diff
@@ -0,0 +1,47 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TRACKRECO")
+
+from FWCore.ParameterSet.VarParsing import VarParsing
+options = VarParsing ('analysis')
+options.parseArguments()
+options.inputFiles = "file:GENSIM.root"
+options.outputFile = "TRACKRECO.root"
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('FastSimulation.Configuration.Geometries_MC_cff')
+process.load('Configuration.StandardSequences.MagneticField_38T_PostLS1_cff')
+process.load('FastSimulation.Configuration.FamosSequences_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+
+from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_mc', '')
+
+process.source = cms.Source(
+    "PoolSource",
+    fileNames = cms.untracked.vstring(options.inputFiles)
+    )
+
+process.generalTracks = cms.EDAlias(
+    generalTracksBeforeMixing = cms.VPSet( 
+        cms.PSet(type=cms.string('floatedmValueMap')),
+        cms.PSet(type=cms.string('recoTracks')),
+        cms.PSet(type=cms.string('recoTrackExtras')),
+        cms.PSet(type=cms.string('TrackingRecHitsOwned')),
+        )
+    )
+
+process.output = cms.OutputModule(
+    "PoolOutputModule",
+    fileName = cms.untracked.string(options.outputFile),
+    outputCommands = cms.untracked.vstring( 
+        ('keep *_*_*_TRACKRECO'),
+        ('drop *_generalTracksBeforeMixing_*_*')
+        )
+    )
+
+
+process.path = cms.Path(process.trackReco)
+process.endpath = cms.EndPath(process.output)
+process.schedule = cms.Schedule([process.path,process.endpath])
```

Comment:
@lveldere 
is this test run automatically now? if yes, be aware that this test depends on the previous one!